blocksync/blocksync.go

```diff
@@ -112,6 +119,9 @@ func NewBlockSyncer(
 		worker:                newSyncWorker(chain.ChainID(), cfg, bsCfg.unicastHandler, bsCfg.neighborsHandler, buf),
 		processSyncRequestTTL: cfg.BlockSync.ProcessSyncRequestTTL,
 	}
+	bs.syncStageTask = routine.NewRecurringTask(bs.syncStageChecker, time.Second*5)
```

Comment:
don't use fixed time.Second*5
use cfg.DardanellesBlockInterval, which is current block time